{
  "FaultLabel": "Resolved power cut",
  "Type": "ResolvedFault",
  "Icon": "icon-success",
  "ShowViewOnMapCTA": false,
  "UnderReview": false,
  "AdditionalFaultInfo": "Good news! The electricity in your area has been restored. The power cut was caused by an unexpected incident on our underground cable that provides your supply.",
  "CTAs": [],
  "faultNumber": "40785462",
  "multipleFaultNumbers": null,
  "date": "2020-11-13T18:27:27",
  "region": "Colne",
  "faultType": "Resolved Fault",
  "faultStatus": "Resolved",
  "previousFaultStatus": "Resolved",
  "consumersOff": 0,
  "estimatedTimeOfRestoration": null,
  "estimatedTimeOfRestorationMajority": "2020-11-13T18:27:27",
  "information": "",
  "actualTimeOfRestoration": "2020-11-13T18:27:27",
  "outageCentrePoint": {
    "lat": 53.847612,
    "lng": -2.192776
  },
  "outageLocations": null,
  "AffectedPostcodes": " BB8 8JD, BB8 8JP, BB8 8JR, BB8 8JS, BB8 8JW, BB8 8LA, BB8 8LF",
  "EtrMUnderReview": true,
  "WebTMSFaultType": "U/GM"
}